tests/components/wemo/entity_test_helpers.py

```diff
@@ -136,6 +138,19 @@ async def test_avaliable_after_update(
     assert hass.states.get(wemo_entity.entity_id).state == STATE_ON
 
 
+async def test_turn_off_state(hass, wemo_entity, domain):
+    """Test that the device state is updated after turning off."""
+    await async_setup_component(hass, domain, {})
```

Comment:
Why do we set up the platform integration? Setting up the wemo integration should be enough.

Reply:
Thanks! Remove in 863a85d